README.md

```diff
@@ -1,75 +1,79 @@
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML5.
+- CSS.
+- JavaScript
+- Visual Studio Code.
+- Git
+- GitHub
+- Webpack
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+If you want to clone this project:
+- Go to GitHub repository
+- Click on the green button that say 'Code'
+- Copy the SSH link
+- On your local machine, type the command git clone and paste the SSH link
 
 
-To get a local copy up and running follow these simple example steps.
-
 ### Prerequisites
+- Code editor
+- Knowledge in HTML
+- Knowledge in Javascript (ES6)
+- Handle on local repository Git
 
 ### Setup
+- Clone the repository
+- Change directory into the project folder
+- Install dependencies
+- Open project with text editor to view changes
+- Run the code using [live server](https://www.google.com/search?client=safari&rls=en&q=live+server&ie=UTF-8&oe=UTF-8)
 
 ### Install
+Clone this repository (you must have [Git Installed](https://github.com/git-guides/install-git))
 
-### Usage
-
-### Run tests
+`git clone git@github.com:robinbenitezmora/ToDoList-Webpack.git`
 
-### Deployment
+Then, move to the newly created folder `ToDoList-Webpack` using this command
 
+`cd ToDoList-Webpack`
 
+Then, install dependencies using this command
 
-## Authors
-
-👤 **Author1**
+Then, install dependencies using this command
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+To open the project in Visual Studio Code type the following command inside the _books_with_es6_ folder
 
-👤 **Author2**
+`code .`
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+In order to visualize the project in your browser either open the index.html file directly from the file explorer or consider using a [live server for development ](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
 
-## 🤝 Contributing
+## Authors
 
-Contributions, issues, and feature requests are welcome!
+👤 **Robin Benitez Mora**
 
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@robinbenitezmora](https://github.com/robinbenitezmora)
+- Twitter: [@Mecanico_Robin](https://twitter.com/mecanico_robin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/robin-benitez-mora-manizales)
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+-I thank Microverse for its documentation, and my mentor, David Verde for his support and guidance.
```

Comment:
- [ ] I would also recommend adding the `Contributing` section with a link to the issue page:

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```